packages/rum-core/src/browser/performanceCollection.spec.ts

```diff
@@ -18,6 +18,12 @@ describe('rum initial document resource', () => {
     retrieveInitialDocumentResourceTiming((timing) => {
       expect(timing.entryType).toBe('resource')
       expect(timing.duration).toBeGreaterThan(0)
+
+      // generate a performance entry like structure
+      const toJsonTiming = timing.toJSON()
+      expect(toJsonTiming.entryType).toEqual(timing.entryType)
+      expect(toJsonTiming.duration).toEqual(timing.duration)
+      expect((toJsonTiming as any).toJSON).toBeUndefined()
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏It seems nice but maybe not worth it, let me know what you think